tools/webtransport/h3/webtransport_h3_server.py

```diff
@@ -35,11 +36,105 @@
 _doc_root: str = ""
 
 
+class CapsuleType(IntEnum):
```

Comment:
This file is getting longer. How about putting Capsule related classes and definitions into a separate file, like capsule.py?

Reply:
Done.